numpy/fft/__init__.py

```diff
@@ -128,11 +128,14 @@
 Normalization
 -------------
 
-The default normalization has the direct transforms unscaled and the inverse
-transforms are scaled by :math:`1/n`. It is possible to obtain unitary
-transforms by setting the keyword argument ``norm`` to ``"ortho"`` (default is
-`None`) so that both direct and inverse transforms will be scaled by
-:math:`1/\\sqrt{n}`.
+The default normalization has the direct (forward) transforms unscaled and the
```

Comment:
Maybe add "backward" here. Maybe also an initial sentence just about the fact that the argument refers to the pair/which direction is transformed.  It reads a bit more difficult than feels necessary to me, but I think its fine to leave for later improvements.  Its not easy to write easy to read docs...

Reply:
ok

Reply:
Thanks, will merge once the tests run through (maybe minus the azure tests if they are acting up again).